src/components/BookList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
```

Comment:
- [ ] It would be better if we create the Book component and render it in the BookList component

Reply:
Hi @addod19, 

Thank you for reviewing my work and your suggestions. I worked on it. 

Cheers!